go.mod

```diff
@@ -5,9 +5,9 @@ go 1.13
 require (
 	github.com/golang/mock v1.3.1
 	github.com/magefile/mage v1.9.0
-	github.com/mattermost/mattermost-server v0.0.0-20190417144445-84a59ddb3928
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20191129165124-8dda3d06b7a2
```

Comment:
This seems to be pointing to commit `8dda3d06b7a2`, but I can't find it in `master`. If this is the `master`'s PR commit, we need to remember to come back and change this pointer when it gets merged.

EDIT: Saw the `do not merge` label on the PR :)

Reply:
Changed to point to v5.18, that was a commit pointing to a closed PR where I introduced a new command `SetConfig` that is not needed anymore because we have `UpdateConfig`